src/config.js

```diff
@@ -0,0 +1,6 @@
+const BASE_URL = '10.58.52.91:3000';
+
+export const API = {
+  kakaoLogin: `http://${BASE_URL}/users/kakaologin`,
+  kakaoAuthToken: 'https://kauth.kakao.com/oauth/token',
+};
```

Comment:
```suggestion
const BASE_URL = 'http://10.58.52.91:3000';

export const API = {
  kakaoLogin: `{BASE_URL}/users/kakaologin`,
  kakaoAuthToken: 'https://kauth.kakao.com/oauth/token',
};
```
BASE URL에 http 프로토콜도 포함시켜주시면 일일이 적지 않아 조금 편해집니다.

Reply:
적용하겠습니다!